tests/PHPUnit/System/expected/test_OneVisitorTwoVisits__DevicesDetection.getType_day.xml

```diff
@@ -37,6 +37,12 @@
 		<logo>plugins/DevicesDetection/images/screens/mobile.gif</logo>
 	</row>
 	<row>
+		<label>Phablet</label>
+		<nb_visits>0</nb_visits>
+		<segment>deviceType==phablet</segment>
+		<logo>plugins/DevicesDetection/images/screens/unknown.gif</logo>
```

Comment:
Ok makes sense - maybe we can create an issue for this design project and I'll see if designers could help with icons (we also discuss to maybe use font awesome which may have icons for some of those devices #7618)